src/component/page/gjennomforing-detaljer/deltaker-oversikt/deltaker-tabell/DeltakerTabell.tsx

```diff
@@ -0,0 +1,72 @@
+import { CheckboxGroup, Table } from '@navikt/ds-react'
+import React, { useEffect, useState } from 'react'
+import { TiltakDeltaker } from '../../../../../api/data/deltaker'
+import { Sortering } from '../../../../../utils/sortering-utils'
+import { TabellBody } from './TabellBody'
+import { TabellHeader } from './TabellHeader'
+
+interface Props {
+	deltakere: TiltakDeltaker[]
+	sortering: Sortering | undefined
+	onSortChange: (v: string | undefined) => void
+	visCheckBox?: boolean
+	onChange?: (val: string[]) => void
+}
+
+export const DeltakerTabell = ({
+	deltakere,
+	sortering,
+	onSortChange,
+	visCheckBox,
+	onChange,
+}: Props) => {
+	const [ markerAlle, setMarkerAlle ] = useState(false)
+	const [ markerte, setMarkerte ] = useState<string[]>([])
+
+	useEffect(() => {
+		if (onChange !== undefined) {
+			onChange(markerte.filter(v => v !== '*'))
+		}
+	}, [ markerte, onChange ])
+
+	const handleChange = (val: string[]) => {
```

Comment:
dette skjønte jeg svert lite av.. Hva er formålet med alle disse endringene på deltakerlista?

Reply:
Det er for bulk endringer, for å controllere sjekkboksene i lista. 
Siden det fortsatt er litt uklart for meg om vi skal forkaste det helt eller ikke har jeg latt det ligge. Ideelt sett ville jeg hatt dette commitet også fjernet om man vil, så kan det hentes tilbake senere. Men jeg kan fjerne alt som har med checkboks, hvis du mener det er best 👍